src/platforms/react-native/upload-debug.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Debug Symbols
+sidebar_order: 96
+description: "Learn about using ProGuard to upload the ProGuard/R8 mappings, source bundle, and native symbols Sentry."
```

Comment:
I may be wrong, but this description doesn't seem to really match what's in the content. 

```suggestion
description: "Learn about different methods for providing debug symbols to Sentry."
```

Reply:
@imatwawana mm I guess it can be improved, but the suggestion is also not ideal.

There are different type of "debug symbols", there are also different ways to upload it, what's about:

> Learn about using the Sentry Android Gradle Plugin to upload the ProGuard/R8 mappings, source bundle, and debug symbols to Sentry.

Reply:
The intro to this page includes this sentence: 
 "We offer a range of methods to provide Sentry with debug symbols." My expectation is that you're about to tell me about a range of methods. If you're talking about a range of methods (which it seems like you are because you talk about the out of the box option), then the description of the page should reflect that, no? If this is really just about ProGuard, then we probably need to remove that sentence or follow that with "but on this page we're only going to talk about ProGuard 8" or whatever makes sense.

Reply:
@imatwawana mm, this sentence was written years ago, don't even remember by whom, but I will try to describe what's supposed to be now.

We have to upload Proguard/R8 mapping file (for obfuscated Java stack traces)
We have to upload debug symbols (for C+/C++ stack traces), the source bundle is an addition to that.

We can do that automatically using the Sentry Android Gradle Plugin.
We can do that via sentry-cli

to be honest I've even suggested renaming this page https://github.com/getsentry/sentry-docs/issues/4395 a few days ago, because its not even about "Proguard" anymore.

Reply:
Okay, gotcha. Is the point of this to tell users about both methods or just to highlight the plugin?

Reply:
ideally both, but highlighting the plugin first, since not everyone is able to use the plugin (the majority is tho), that's why there's an alternative, which is thru sentry-cli

Reply:
I've rewritten it, please have a second look @imatwawana, and sorry for the ping pong.

Reply:
Looks great!!